app/controllers/home_controller.rb

```diff
@@ -1,6 +1,6 @@
 class HomeController < ApplicationController
   def index
-      @pg_title = "Home | Haxxor News"
-      @articles = Article.order(id: :desc)
+		@pg_title = "Home | Haxxor News"
```

Comment:
watch out for the rebase here, think you updated this to be `@page_title` right?